sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -38,82 +37,147 @@
 
 
 class SdkHarness(object):
+  REQUEST_METHOD_PREFIX = '_request_'
+
+  def __init__(self, control_address, pipeline_options=None):
+    def _get_worker_count(pipeline_options):
+      pipeline_options = pipeline_options.get(
```

Comment:
Can we move the code related to cleaning up `pipeline_options` to sdk_worker_main.py, where `PIPELINE_OPTIONS` environment variable is parsed.